frontend/src/routes/Home.svelte

```diff
@@ -1,22 +1,178 @@
 <script lang="ts">
-</script>
+  import yup from 'yup';
+  import svelteFormsLib from 'svelte-forms-lib';
+  import { _ } from 'svelte-i18n';
+  import { navigate } from 'svelte-routing';
+  import Spinner from 'svelte-tailwind-widgets/Spinner.svelte';
+  import { createConfiguration, DefaultApi, ServerConfiguration } from 'jimmi-api-client/mod.ts';
+  import config from '../config.ts';
+
+  interface IJimmiSettings {
+    domain: string;
+    room: string;
+    password: string;
+  }
+
+  const domainRegex = new RegExp(
+    /^([a-zA-Z](?:(?:([a-zA-Z]|[0-9]|-)+)?[a-zA-Z]|[0-9])?\.)*[a-zA-Z](?:(?:([a-zA-Z]|[0-9]|-)+)?[a-zA-Z]|[0-9])?$/
+  );
+
+  const jitsiRoomRegex = new RegExp(/^[^?&:"'%#]+$/); // only given characters are excluded
+
+  const { form, errors, handleChange, handleSubmit } = svelteFormsLib.createForm({
+    initialValues: {
+      domain: '',
+      room: '',
+      password: '',
+    },
+    validationSchema: yup.object().shape({
+      domain: yup.string().trim().max(63).matches(domainRegex),
+      room: yup.string().trim().min(1).matches(jitsiRoomRegex).required(),
+      password: yup.string().optional(),
+    }),
+    onSubmit: (settings: IJimmiSettings) => {
+      if (!settings.domain) {
+        settings.domain = config.defaultJitsiUrl;
+      }
+      isLoading = true;
+      createJimmiInstance(settings);
+    },
+  });
+
+  /**
+   * Creates a new Jimmi instance and redirects the user to the corresponding dashboard
+   *
+   * @param settings - The Jimmi instance settings
+   */
+  async function createJimmiInstance(settings: IJimmiSettings) {
+    const apiConfig = createConfiguration({
+      baseServer: new ServerConfiguration(config.apiBaseUrl),
+    });
+    const jimmiApi = new DefaultApi(apiConfig);
+    const instance = await jimmiApi.instancesPost();
+    // ToDo: Join room etc, once API is ready for that
```

Comment:
ToDo Statement o.O

Reply:
API Client wasn't ready, back in the day :stuck_out_tongue: 
Fixed in 153909a99b4f04b008a516775aac38132702bed3